### PR TITLE
Push Encryptonize container images to GitHub

### PR DESCRIPTION
**Pull request by kim-cc**
### Description
Push Encryptonize EAAS and object container images to Github packages.
We need to images available for the Sample deployment to fully work.

### Type(s) of Change (Split the PR if you check many)
- [X] Added user feature
- [ ] Added technical feature
- [ ] Added tests
- [ ] Refactor
- [ ] Bugfix
- [ ] Updated documentation
- [ ] Kubernetes changes
- [X] CI/CD workflow changes

### Testing Checklist
- [ ] I have added/updated unit tests for functional changes.
- [ ] I have added/updated integration tests for changes that affect dependent systems.
- [ ] I have added/updated end-to-end tests for feature changes.

### General Checklist
- [X] I have pulled in the latest changes from master.
- [ ] I have run `make lint`.
- [ ] I have successfully run `make tests`.
- [X] I have updated relevant CI/CD workflows.
- [ ] I have updated relevant Kubernetes manifests/scripts.
- [ ] I have updated/added relevant (internal and external) documentation (readme, doc comments, etc).
- [ ] I have updated the dependency map to reflect my changes.
- [ ] I have added the license to new source code files.
- [X] I have spent some time looking over the full diff before creating this PR.

### CODE REVIEW

File: integrations/eaas/makefile [hunk 1]
```diff
@@ -59,7 +63,11 @@ docker-build: git-commit=$(shell git rev-list -1 HEAD)
 docker-build: git-tag=$(shell git tag --points-at HEAD)
 docker-build:  ## Build the Encryptonize EAAS docker image
 	DOCKER_BUILDKIT=1 \
-	docker build --build-arg COMMIT="${git-commit}" --build-arg TAG="${git-tag}" -t encryptonize-eaas -f Dockerfile ../..
+	docker build --build-arg COMMIT="${git-commit}" --build-arg TAG="${git-tag}" -t ${image_name}:${image_tag} -f Dockerfile ../..
+
+.PHONY: docker-push
+docker-push:
```
Comment (philip-cc) [inline integrations/eaas/makefile:69]: Maybe make `docker-build` a dependency?

File: .github/workflows/integrations-build.yaml [hunk 2]
```diff
@@ -88,6 +88,33 @@ jobs:
...
+
+    - name: Log in to the Github container registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: cybot-cc
+        password: ${{ secrets.CYBOT_TOKEN }}
+
+    - name: Push container image
+      if: github.ref_name == 'master'
```
Comment (philip-cc) [inline .github/workflows/integrations-build.yaml:115]: Shouldn't this be at the job level so we skip this job entirely when not on master?

Comment (kim-cc) [reply .github/workflows/integrations-build.yaml:115]: Yes it should, didn't think about that

Comment (kim-cc) [reply .github/workflows/integrations-build.yaml]: Fixed

File: integrations/eaas/makefile (hunk 1 above)
Comment (kim-cc) [reply integrations/eaas/makefile]: Good point, fixed

**Review by philip-cc — APPROVED** [review]

**Pull request merged**